Exercise_1/Program.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.SqlServer.Server;
+using Newtonsoft.Json;
+
+namespace Exercise_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
The program should enter the `email to`, `email content` and the send the email

Reply:
Done

Reply:
Done cái nồi

Reply:
@@ cái em nhập vào thêm vào rồi mà , còn cái send với setcontent chưa sửa :D